lib/fbe/middleware/logging_formatter.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+# MIT License
+#
+# Copyright (c) 2024 Zerocracy
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+require 'faraday'
+
+# Faraday logging formatter show verbose log for only error response
+class Fbe::Middleware::LoggingFormatter < Faraday::Logging::Formatter
+  AUTHORIZATION_FILTER = [/(Authorization: )([^&]+)([^&]{5})/, '\1********\3'].freeze
```

Comment:
@Yegorov Could you explain why a fixed number of 8 asterisks was chosen? Is there a specific reason for this choice, or is it just a convention?

Reply:
@Suban05 Fixed count - for security reason so that they do not know the real size of token. 8 - only just number (maybe change to other), this idea get from [docs](https://lostisland.github.io/faraday/#/middleware/included/logging?id=filter-sensitive-information)

The log looks like this now:
```
3.3.4 :001 > resp = octo.user(11)                                                                                                           
E: GET https://api.github.com/user/11                                                                                                       
E: User-Agent: "Octokit Ruby Gem 9.1.0"                                                                                                     
   Accept: "application/vnd.github.v3+json"                                                                                                 
   Content-Type: "application/json"                                                                                                         
   Authorization: ********ReSk"                                                                                                             
E: Status 404                                                                                                                               
E: date: "Sat, 21 Sep 2024 18:32:25 GMT"                                                                                                    
   content-type: "application/json; charset=utf-8"                                                                                          
   github-authentication-token-expiration: "2025-08-25 00:00:00 +0300"                                                                      
   x-github-media-type: "github.v3; format=json"                                                                                            
   x-github-api-version-selected: "2022-11-28"                                                                                              
   x-ratelimit-limit: "5000"                                                                                                                
   x-ratelimit-remaining: "4999"                                                                                                            
   x-ratelimit-reset: "1726947145"                                                                                                          
   x-ratelimit-used: "1"                                                                                                                    
   x-ratelimit-resource: "core"
   access-control-expose-headers: "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"
   access-control-allow-origin: "*" 
   strict-transport-security: "max-age=31536000; includeSubdomains; preload"
   x-frame-options: "deny"
   x-content-type-options: "nosniff"
   x-xss-protection: "0"
   referrer-policy: "origin-when-cross-origin, strict-origin-when-cross-origin"
   content-security-policy: "default-src 'none'"
   vary: "Accept-Encoding, Accept, X-Requested-With"
   content-encoding: "gzip"
   transfer-encoding: "chunked"
   server: "github.com"
   x-github-request-id: "E50E:0EBA:6F19B1:708E7F:66EF1139"
E: {"message":"Not Found","documentation_url":"https://docs.github.com/rest","status":"404"}
```